docs/index.rst

```diff
@@ -30,7 +30,7 @@ Table of Contents
 - `Quickstart`_
 - `What is the Ray provider?`_
 - `Components`_
-- `Contact the Devs`_
+- `Getting Involved`_
```

Comment:
No strong opinion but I like to name `Community` this section

Reply:
Lets do it in a separate PR